web_widget_mpld3_chart/static/src/js/web_widget_mpld3_chart.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('web_widget_mpld3_chart', function (require) {
+"use strict";
+
+    var fieldRegistry = require('web.field_registry');
+    var AbstractField = require('web.AbstractField');
+
+    var Mpld3ChartWidget = AbstractField.extend({
+        start: function() {
```

Comment:
_renderEdit will be the right place.

Reply:
@MeetKD tried your suggestion and did not work. The current implementation drives the correct rendering.